config/__init__.py

```diff
@@ -21,11 +21,6 @@ class ProfileWarning(Warning):
     (and other sub-modules use it)"""
     pass
 
-
-user = 'NOT_SPECIFIED'
-DEBUG = False
-debug_token = 'debug-token'
-
 try:
```

Comment:
Wait I meant the `profile` import, not the fields above. If you pip install this, you can't just update the `profile.py` locally easily.